framework/rbac/BaseManager.php

```diff
@@ -220,4 +220,15 @@ protected function executeRule($user, $item, $params)
             throw new InvalidConfigException("Rule not found: {$item->ruleName}");
         }
     }
+
+    /**
+     * Check that there no any roles in user's role assignments and in default roles
+     *
+     * @param Assignment[] $assignments list of user's role assignments
+     * @return bool true if empty
```

Comment:
`@return bool whether $assignments array is empty and [[defaultRoles]] are empty as well`